src/Handler.cpp

```diff
@@ -224,7 +260,7 @@ namespace mediasoupclient
 		catch (std::exception& error)
 		{
 			// Panic here. Try to undo things.
-			transceiver->SetDirection(webrtc::RtpTransceiverDirection::kInactive);
+			//transceiver->SetDirection(webrtc::RtpTransceiverDirection::kInactive);
```

Comment:
Why is this commented? If it doesn't make sense let's just remove the line. But why?

Reply:
just mark it,maybe i will change lower version of webrtc

Reply:
I don't understand. How does lower version of webrtc affect here? My question is: why is that line commented in your PR?

Reply:
I'm careless. The old interface has been deprecated：
RTC_DEPRECATED virtual void SetDirection(
      RtpTransceiverDirection new_direction);